astropy/coordinates/angles.py

```diff
@@ -173,15 +173,15 @@ def hour(self):
     def hms(self):
         """
         The angle's value in hours, as a named tuple with ``(h, m, s)``
-        members.  (This is a read-only property.)
+        members.  (This is a read-only property.).
```

Comment:
This extra `.` is wrong... I would just remove "This is a read-only property"

Reply:
✅. I've removed "This is a read-only property"